index.js

```diff
@@ -5,14 +5,42 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "Magenta"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
 
+    let profileDiv = document.createElement('div')
+    let img = document.createElement('img')
+    let name = document.createElement('h2')
+    let nickname = document.createElement('em')
 
-/***** Deliverable 4 *****/
+    profileDiv.id = ("profile")
+    img.src = traveler.photo
+    img.alt = traveler.name
+    name.textContent = traveler.name 
+    nickname.textContent = traveler.nickname
+    
+    let profiles = document.querySelector("#profile")
+    profileDiv.append(img, name, nickname)
+    profiles.append(profileDiv)
 
 
+/***** Deliverable 4 *****/
+let animals = document.querySelector("#animals")
+    traveler.animalSightings.forEach(function(animal){
+        animals.insertAdjacentHTML('beforeend', `
+        
+            <li data-id= "${animal.id}">
+            <p>{animal.description}</p>
+            <img src="${animal.photo}" alt="${animal.species}"/>
+            <a href="${animal.link}" target="_blank">Here's a video about the ${animal.species} species!</a>
+        </li>  `)
```

Comment:
Great work on this! `insertAdjacentHTML` is a nice approach to creating DOM elements from strings. It's also more flexible than using `innerHTML +=`, since that destroys existing DOM elements and overwrites them. Good find!